src/Components/PaymentDropDownField.res

```diff
@@ -62,26 +88,52 @@ let make = (
           style={ReactDOMStyle.make(
             ~background=disabled ? disbaledBG : themeObj.colorBackground,
             ~opacity=disabled ? "35%" : "",
-            ~padding=themeObj.spacingUnit,
+            ~padding="11px 20px 11px 11px",
```

Comment:
Why are we hardcoding the spacing instead of taking it from spacingUnit?

Reply:
spacing unit is to space between the components, and we are using our own component which needs to be given custom padding all all sides due to the overlapping of the Icon and Placeholder